AppDevAnnouncements/Classes/NotificationViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  NotificationViewController.swift
+//  AppDevAnnouncements
+//
+//  Created by Gonzalo Gonzalez on 3/11/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol NotificationDelegate: class {
+    func present(_ notificationViewController: NotificationViewController)
+}
+
+class NotificationViewController: UIViewController {
+
+    private var notificationView: NotificationView!
+
+    private var visualUrl : String!
+    private var subject : String!
+    private var body : String!
+    private var ctaText : String!
+    private var ctaAction : String!
+    private var delegate : NotificationDelegate!
+
+    convenience init(visualUrl: String, subject: String, body: String, ctaText: String, ctaAction: String, delegate: NotificationDelegate) {
+        self.init()
+        self.visualUrl = visualUrl
+        self.subject = subject
+        self.body = body
+        self.ctaText = ctaText
+        self.ctaAction = ctaAction
+        self.delegate = delegate
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .clear
+
+        notificationView = NotificationView(dismissFunc: #selector(dismissPopup), visualUrl: visualUrl, subject: subject, body: body, ctaText: ctaText, actionFunc: #selector(openSite), target: self)
+        notificationView.layer.cornerRadius = 10
+        notificationView.clipsToBounds = true
+        view.addSubview(notificationView)
+        
+        setupConstraints()
+        presentNotification()
+    }
+    
+    private func setupConstraints(){
```

Comment:
nitty gritty but please add space between () { ?? just to keep things consistent!

Reply:
please tear apart my pull request with nitty grittiness i want my code to be look and function perfectly @lucyxubroad